### PR TITLE
Add Restruct with policy selector in add SA modal

### PR DESCRIPTION
**Pull request by dvaldivia**
Adds a toggle to show/hide the policy field for creating a service account
Screen Shot 2021-04-13 at 2 58 34 PM
Screen Shot 2021-04-13 at 2 58 31 PM

### CODE REVIEW

File: portal-ui/src/screens/Console/Account/AddServiceAccount.tsx [hunk 1]
```diff
@@ -106,8 +111,8 @@ const AddServiceAccount = ({
           addServiceAccount(e);
         }}
       >
-        <Grid container>
-          <Grid item xs={12} className={classes.formScrollable}>
+        <Grid container className={classes.formScrollable}>
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Account/AddServiceAccount.tsx:114]: This class is not meant to be applied in a grid container as it adds an extra padding at the bottom of the container.

Screen Shot 2021-04-13 at 17 33 11

**Review by bexsoft — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Account/AddServiceAccount.tsx (hunk 1 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Account/AddServiceAccount.tsx:114]: fixed

**Comment by dvaldivia** [comment]
Screen Shot 2021-04-13 at 3 37 33 PM

**Review by bexsoft — APPROVED** [review]
LGTM, Tested OK

**Review by Alevsk — DISMISSED** [review]

**Review by bexsoft — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Pull request merged**